utils/scheduler.py

```diff
@@ -0,0 +1,29 @@
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from utils.core import get_random_quote
+import logging
+from logging.config import fileConfig
+
+fileConfig('../utils/logging_config.ini')
+logger = logging.getLogger('discord')
+
+
+def init_message_scheduler(config, client):
+
```

Comment:
It would be preferable to have a short comment string explaining what happens here. 

```python
"""Pass in a configuration and scheduled jobs will be added accordingly."""
```

Or something to that effect. Of course, it isn't strictly necessary since the function is pretty clear, so I will leave it up to you.